<launch>
  <arg name="people_topic" default="/tracked_people"/>
  <arg name="goal_param"  default=""/>
  <arg name="target_frame"  default="robot_map"/>

  <node name="roscausal_human" pkg="roscausal_human" type="human_state.py" output="screen">
    <param name="people_topic" type="string" value="$(arg people_topic)"/>
    <param name="goal_param" type="string" value="$(arg goal_param)"/>
    <param name="target_frame" type="string" value="$(arg target_frame)"/>
  </node>

</launch>
